CMakeLists.txt

```diff
@@ -20,7 +20,15 @@
 # SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
 ################################################################################
 
-cmake_minimum_required(VERSION 3.4 FATAL_ERROR)
+# CMAKE_VERSION is not defined before 2.6.3.
+if ((CMAKE_MAJOR_VERSION LESS 3) OR (CMAKE_VERSION VERSION_LESS "3.12"))
+	cmake_minimum_required(VERSION 3.4 FATAL_ERROR)
```

Comment:
Why not update the OpenJ9 min to 3.5 anyway, since we don't need to support 3.4.

Reply:
We could, but there's no need. It should still work with 3.4 even if nobody ever tries.

Reply:
Depends on the platform. I'll save the merging for next week.